### PR TITLE
fix: add support for parent paths without operations

### PR DESCRIPTION
**Pull request by childish-sambino**
Needed for fluent-style libraries which depend on parent paths without operations.

### CODE REVIEW

File: examples/node/lib/rest/api/V2010.ts [hunk 1]
```diff
@@ -29,10 +31,16 @@ export default class V2010 extends Version {
     }
 
     protected _accounts?: AccountListInstance;
+    protected _credential?: CredentialListInstance;
```
Comment (mattcole19) [inline examples/node/lib/rest/api/V2010.ts:34]: Should this be credentials?

Comment (childish-sambino) [reply examples/node/lib/rest/api/V2010.ts:34]: nerp: https://github.com/twilio/twilio-oai-generator/blob/578de68244482fa90fdaa620ec7fd41543b509e2/examples/twilio_api_v2010.yaml#L124

File: examples/node/spec/unit/credential.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+import nock from 'nock';
+import Twilio from '../../lib/rest/Twilio';
+
+describe('credential', () => {
+    const twilio = new Twilio();
+
+    it('should update an aws credential', () => {
```
Comment (beebzz) [inline examples/node/spec/unit/credential.spec.ts:8]: Should we also have a delete test?

**Review by mattcole19 — APPROVED** [review]

File: examples/node/spec/unit/credential.spec.ts (hunk 2 above)
Comment (childish-sambino) [reply examples/node/spec/unit/credential.spec.ts:8]: We _could_, I was just focusing on making sure we could do _anything_ against this endpoint.

**Pull request merged**